lib/tasks/lint.rake

```diff
@@ -1,5 +1,10 @@
 desc "Run all linters"
 task lint: :environment do
+  if Rails.env.development?
+    sh "bundle exec erblint --lint-all --autocorrect"
```

Comment:
fair point, I was not sure about this. I guess we can keep it consistent like rubocop, only reason i added --autocorrect was I was not sure in case someone not sure which flag to use.